sport/app/conf/SportConfiguration.scala

```diff
@@ -8,7 +8,7 @@ object SportConfiguration {
   import GuardianConfiguration._
 
   object pa {
-    lazy val footballHost = PaClientConfig.baseUrl
+    lazy val footballHost = "http://football-api.gu-web.net/v1.5"
```

Comment:
This host is used by our online docs and troubleshooter, so I'm not bothering to put a switch on it.